go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/dgraph-io/badger/v3 v3.2103.2
 	github.com/emersion/go-imap v1.2.1-0.20220429085312-746087b7a317
 	github.com/emersion/go-imap-uidplus v0.0.0-20200503180755-e75854c361e9
-	github.com/emersion/go-mbox v1.0.2
+	github.com/emersion/go-mbox v1.0.3
```

Comment:
I'd suggest you just use the full dependency `github.com/ProtonMail/go-mbox` here rather than replacing it -- just change the module name in that repo to match.